common/lib/xmodule/xmodule/js/spec/video/video_player_spec.js

```diff
@@ -1011,6 +1014,39 @@ function(VideoPlayer, HLS) {
                 );
             });
         });
+
+        describe('Overlay Play Button', function() {
+            var playButtonOverlaySelector = '.video-wrapper .btn-play.fa.fa-youtube-play.fa-2x';
+            beforeEach(function() {
+                state = jasmine.initializePlayer();
+            });
+
+            it('shows the play button after player is ready', function(done) {
+                jasmine.fireEvent(state.videoPlayer.player.video, 'loadedmetadata');
+                jasmine.waitUntil(function() {
+                    return state.videoPlayer.player.getPlayerState() !== STATUS.UNSTARTED;
+                }).then(function() {
+                    expect($(playButtonOverlaySelector)).not.toHaveClass('is-hidden');
+                }).always(done);
+            });
+
+            it('hides the play button on play', function() {
+                $(state.videoPlayer.player.videoEl).trigger('click');  // play
+                expect($(playButtonOverlaySelector)).toHaveClass('is-hidden');
+            });
+
+            it('shows the play button on pause', function() {
+                $(state.videoPlayer.player.videoEl).trigger('click');  // play
+                $(state.videoPlayer.player.videoEl).trigger('click');  // pause
+                expect($(playButtonOverlaySelector)).toHaveClass('is-hidden');
+            });
+
+            it('plays the video when overlay button is clicked', function() {
+                $('.video-wrapper .btn-play').trigger('click');  // play
+                expect($(playButtonOverlaySelector)).toHaveClass('is-hidden');
+                expect(state.videoPlayer.player.getPlayerState()).toEqual(STATUS.PLAYING);
```

Comment:
Is this test needed? 
`expect(state.videoPlayer.player.getPlayerState()).toEqual(STATUS.PLAYING)` can be moved to `it('hides the play button on play' function(){...}`